src/filter.c

```diff
@@ -814,6 +814,10 @@ static int filter_ioctx_ctrl_cb(AVBufferRef *event_ref, void *callback_ctx,
                     sp_frame_fifo_set_max_queued(ctx->in_pads[i]->fifo, len);
             }
         }
+        if ((tmp_val = dict_get(event->opts, "send_eos"))) {
```

Comment:
is there a single place where i can add it? are there other such "standard options"? i can't figure out where to add it so that it applies to all `ioctx_ctrl` callbacks 😕 (unless im actually adding this option to all other components?)